src/pcp.rs

```diff
@@ -240,7 +248,45 @@ where
     /// "leader" share from the others. This is useful, for example, when some of the gadget inputs
     /// are constants used for both proof generation and verification.
     ///
-    /// TODO(cjpatton) Add an example once we have implemented a data type that uses this method.
+    /// ```
+    /// use prio::pcp::types::MeanVarUnsignedVector as MvuVec;
```

Comment:
nit: using an alias like this in a doc example makes it a little harder to understand because the reader might then grep through the crate looking for the `MvuVec` type when what they really want is `MeanVarUnsignedVector`.